packages/blocks/src/api/raw-handling/nested-list-converted.js

```diff
@@ -0,0 +1,87 @@
+function isList( node ) {
+	return node.nodeName === 'OL' || node.nodeName === 'UL';
+}
+
+function createLineBreak( doc ) {
+	return doc.createElement( 'br' );
+}
+
+function getDepth( listNode, depth = 0 ) {
+	if ( isList( listNode.parentElement ) ) {
+		return getDepth( listNode.parentElement, depth + 1 );
+	}
+	return depth;
+}
+
+function createSpace( length ) {
+	return (
+		Array.from( { length } )
+			// eslint-disable-next-line no-unused-vars
+			.map( ( i ) => `\u00A0 \u00A0` )
```

Comment:
Why three spaces?

Reply:
One is hard to spot, and two didn't feel like enough. This produces something closer to the source list.